intro.tex

```diff
@@ -8,7 +8,7 @@ \section{Introduction}
 \item	Encrypt data using strong, approved encryption standards,  following \citeds{NIST.SP.800-171} for \gls{CUI} at non federal organizations. The required use of these \gls{NIST} security standards is limited to the physical security and encryption points. It does not extend to treating the data as \gls{CUI}. The data should \emph{not} be marked as \gls{CUI}.
 \item	Use firewalls and physical security best practices to prevent unauthorized network  access. Documented compliance shall be in accordance with \citeds{NIST.SP.800-171}.
 \item	Delay public release of  focal plane scientific data for an embargo period of at least 80 hours following the  observation.
-Hold engineering and commissioning imaging data for an embargo period of at least 30 days.
+Hold engineering-mode commissioning imaging data for an embargo period of at least 30 days.
```

Comment:
that changes the meaning - this says i can publish on sky images which are not engineering mode ...

Reply:
Maybe "Hold engineering mode images for 30 days and science validation images for regular operations period of 80 hours. 

This could be the regular mode for all of ops, where we modify validation to just regular on sky or some such.

Reply:
Adopted @rblum5 's formulation, with some tweaks